performant-software/Annotation-Studio
# Default to mine tab after document creation

## What this PR does?
1. Default to mine tab after document creation

## How to test?
1. Go to http://cove-staging.herokuapp.com/documents new after signing in
2. Create a new document
3. Check to see if you are on the mine tab
